src/Storages/System/StorageSystemProcesses.cpp

```diff
@@ -50,17 +51,17 @@ NamesAndTypesList StorageSystemProcesses::getNamesAndTypes()
         {"quota_key", std::make_shared<DataTypeString>()},
         {"distributed_depth", std::make_shared<DataTypeUInt64>()},
 
-        {"elapsed", std::make_shared<DataTypeFloat64>()},
-        {"is_cancelled", std::make_shared<DataTypeUInt8>()},
-        {"is_all_data_sent", std::make_shared<DataTypeUInt8>()},
-        {"read_rows", std::make_shared<DataTypeUInt64>()},
-        {"read_bytes", std::make_shared<DataTypeUInt64>()},
-        {"total_rows_approx", std::make_shared<DataTypeUInt64>()},
+        {"elapsed", std::make_shared<DataTypeFloat64>(), "The time in seconds since request execution started."},
+        {"is_cancelled", std::make_shared<DataTypeUInt8>(), "Was query cancelled."},
+        {"is_all_data_sent", std::make_shared<DataTypeUInt8>(), "Was all data sent to the client (in other words query had been finished on the server)."},
+        {"read_rows", std::make_shared<DataTypeUInt64>(), "The number of rows read from the table. For distributed processing, on the requestor server, this is the total for all remote servers."},
+        {"read_bytes", std::make_shared<DataTypeUInt64>(), "The number of uncompressed bytes read from the table. For distributed processing, on the requestor server, this is the total for all remote servers."},
+        {"total_rows_approx", std::make_shared<DataTypeUInt64>(), "The approximation of the total number of rows that should be read. For distributed processing, on the requestor server, this is the total for all remote servers. It can be updated during request processing, when new sources to process become known."},
         {"written_rows", std::make_shared<DataTypeUInt64>()},
         {"written_bytes", std::make_shared<DataTypeUInt64>()},
-        {"memory_usage", std::make_shared<DataTypeInt64>()},
+        {"memory_usage", std::make_shared<DataTypeInt64>(), "Amount of RAM the request uses. It might not include some types of dedicated memory"},
```

Comment:
```suggestion
        {"memory_usage", std::make_shared<DataTypeInt64>(), "Amount of RAM the query uses. It might not include some types of dedicated memory"},
```